crates/script/src/lib.rs

```diff
@@ -199,6 +198,10 @@ pub struct ScriptArgs {
     )]
     pub with_gas_price: Option<U256>,
 
+    /// The CREATE2 deployer address to use, this will override the one in the config.
+    #[arg(long, value_name = "ADDRESS")]
+    pub create2_deployer: Option<Address>,
+
```

Comment:
could we make it part of `EvmArgs` so that it's available for all commands? I believe this is how it's done for all options from `EvmOpts`